doc/md/contributing.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Contributing
+id: contributing
+slug: /contributing
+---
+Atlas is a community project, we welcome contributions of all kinds and sizes!
+
+Here are some ways in which you can help:
+* File well-written and descriptive bug reports or feature requests in the [Issues page](https://github.com/ariga/atlas/issues).
+* Tweet about your experience with Atlas on [Twitter](https://twitter.com), don't forget to mention
+  [@ariga_io](https://twitter.com/ariga_io) and link to [atlasgo.io](https://atlasgo.io) if you do.
+* Write educational content on your personal blog or websites such as [dev.to](https://dev.to) or 
+  [Medium](https://medium.com). If you do, don't hesitate to reach out to us via our
+  [Discord Server](https://discord.com/invite/QhsmBAWzrC) for help proof-reading your text and 
+  using our social-media channels for distributing it to readers.
+* Join our [Discord Server](https://discord.com/invite/QhsmBAWzrC) to answer questions of other users
+  and find out other ways in which you can contribute by talking to the community there!
+* Contribute bug-fixes or new features to the [codebase](https://github.com/ariga/atlas).
+
+### Contributing Code to Atlas
+
+As we are still starting out, we don't have an official code-style or guidelines on composing your
+code. As general advice, give the area of the code that you are modifying and try to keep your code
```

Comment:
That sentence is weird. What does it mean?

Reply:
Modified it a bit to be clearer. I meant that contributors should look around the area they are modifying and try to stick to the style that it's written in.